swayidle/swayidle.1.txt

```diff
@@ -36,20 +36,19 @@ Options
 *-h*::
 	Show help message and quit.
 
-*-i <idle timeout in seconds>*::
-	Turn off the screen after the system have been idle for the specified 
-	number of seconds.
-
-*-l <lock timeout in seconds>*::
-	Lock the screen after the system have been idle for the specified 
-	number of seconds.
-
 *-x <lock command>*::
-	Raise the lockscreen using the lock command, upon systemd sleep events.
+	Raise the lockscreen using the lock command (by executing it in a shell), upon systemd sleep events.
 
 *-d*::
 	Enable debug output.
 
+Sample usage
+------------
+
+swayidle -x \'swaylock -c 000000' timeout 300 exec \'swaylock -c 000000' 
+timeout 600 exec \'swaymsg "output * dpms off"' resume exec \'swaymsg "output *
+dpms on"'
+
```

Comment:
Let's not try to maintain two examples, just stick with the one in the default config